cmd/prysmctl/validator/cmd.go

```diff
@@ -160,12 +160,12 @@ var Commands = []*cli.Command{
 					flags.InteropStartIndex,
 					cmd.GrpcMaxCallRecvMsgSizeFlag,
 					flags.CertFlag,
-					flags.GrpcHeadersFlag,
-					flags.GrpcRetriesFlag,
-					flags.GrpcRetryDelayFlag,
+					flags.GRPCHeadersFlag,
+					flags.GRPCRetriesFlag,
+					flags.GRPCRetryDelayFlag,
 					flags.ExitAllFlag,
 					flags.ForceExitFlag,
-					flags.VoluntaryExitJSONOutputPath,
+					flags.VoluntaryExitJSONOutputPathFlag,
```

Comment:
It's great to have these all organized! Counter proposal: what about removing the `Flag` at the end since these are `flags.<FlagName>` and obviously a flag?
```suggestion
					flags.VoluntaryExitJSONOutputPath,
```
Same applies to all other flags.

Reply:
Since we have the same pattern for the beacon node I would rather have this as a separate PR